<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupReportMonthActionsDialog</class>
 <widget class="QDialog" name="SetupReportMonthActionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="11" column="1" colspan="4">
    <widget class="QFrame" name="frmAge">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblAge">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>с</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeFrom</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
        <property name="value">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="23" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="4">
    <widget class="QCheckBox" name="chkPersonOSDetail">
     <property name="text">
      <string>Детализировать по подразделениям</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="4">
    <widget class="QComboBox" name="cmbSex">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Мужской</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Женский</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QCheckBox" name="chkLocality">
     <property name="text">
      <string>Местность</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="chkSex">
     <property name="text">
      <string>Пол</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="chkAge">
     <property name="text">
      <string>Возраст </string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="4">
    <widget class="QComboBox" name="cmbLocality">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>не учитывать</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>городские</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>сельские</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="21" column="1" colspan="4">
    <widget class="QCheckBox" name="chkShowVisits">
     <property name="text">
      <string>Отображать визиты</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="4">
    <widget class="QComboBox" name="cmbUET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>по договору</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по нормативу</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblUET">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Учитывать УЕТ</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="1" colspan="4">
    <widget class="QCheckBox" name="chkPersonOverall">
     <property name="text">
      <string>Общий по врачам</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="4">
    <widget class="QCheckBox" name="chkPersonDetail">
     <property name="text">
      <string>Отдельно по врачам</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1" colspan="4">
    <widget class="QCheckBox" name="chkPersonMonthDetail">
     <property name="text">
      <string>Детализировать по месяцам</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1" colspan="4">
    <widget class="QCheckBox" name="chkPersonDateDetail">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Детализировать по датам</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbAssistant">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkAssistant">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ассистент</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QComboBox" name="cmbClass">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Статус</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Диагностика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лечение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прочие мероприятия</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Назначено</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkClass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Класс</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="CActionTypeComboBox" name="cmbActionTypeGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkActionTypeGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="CActionStatusComboBox" name="cmbStatus">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkFinance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbFinance">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Период</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="chkTissueType">
     <property name="text">
      <string>Тип биоматериала</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="btnSpecialityList">
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkPerson">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Исполнитель</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QLabel" name="lblSpecialityList">
     <property name="text">
      <string>не задано</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkOrgStructure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbPerson">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbTissueType">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="4">
    <widget class="CARMSContractTreeFindComboBox" name="cmbContract"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblContract">
     <property name="text">
      <string>Договор</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>Events/ActionTypeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionStatusComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Events/ActionStatus.h</header>
  </customwidget>
  <customwidget>
   <class>CARMSContractTreeFindComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/ContractFindComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>chkFinance</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>chkClass</tabstop>
  <tabstop>cmbClass</tabstop>
  <tabstop>chkActionTypeGroup</tabstop>
  <tabstop>cmbActionTypeGroup</tabstop>
  <tabstop>chkStatus</tabstop>
  <tabstop>cmbStatus</tabstop>
  <tabstop>chkOrgStructure</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>chkPerson</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>chkAssistant</tabstop>
  <tabstop>cmbAssistant</tabstop>
  <tabstop>chkTissueType</tabstop>
  <tabstop>cmbTissueType</tabstop>
  <tabstop>chkAge</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>chkSex</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>chkLocality</tabstop>
  <tabstop>cmbLocality</tabstop>
  <tabstop>chkPersonOSDetail</tabstop>
  <tabstop>chkPersonOverall</tabstop>
  <tabstop>chkPersonDetail</tabstop>
  <tabstop>chkPersonMonthDetail</tabstop>
  <tabstop>chkPersonDateDetail</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupReportMonthActionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupReportMonthActionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFinance</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbFinance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkClass</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbClass</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkActionTypeGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbActionTypeGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStatus</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbStatus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrgStructure</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbOrgStructure</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPerson</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbPerson</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAssistant</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAssistant</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkTissueType</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbTissueType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAge</sender>
   <signal>toggled(bool)</signal>
   <receiver>frmAge</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSex</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbSex</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPersonDetail</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkPersonDateDetail</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLocality</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbLocality</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
